java/src/main/java/org/rocksdb/RocksDB.java

```diff
@@ -64,8 +64,7 @@ public static void loadLibrary() {
         NativeLibraryLoader.getInstance().loadLibrary(tmpDir);
       } catch (IOException e) {
         libraryLoaded.set(LibraryState.NOT_LOADED);
-        throw new RuntimeException("Unable to load the RocksDB shared library",
```

Comment:
I guess this is not meant to be in this PR?

Reply:
Oh this is actually in the base PR. Thanks for catching this @adamretter!